source/jormungandr/jormungandr/utils.py

```diff
@@ -494,6 +494,30 @@ def get_pt_object_from_json(dict_pt_object, instance):
     return pt_object
 
 
+def transform_entrypoint(dict_pt_object, uri):
+    if not dict_pt_object:
+        return None
+    if MAP_STRING_PTOBJECT_TYPE.get(dict_pt_object.get("embedded_type")) != type_pb2.ADDRESS:
+        return dict_pt_object
+    first_within_zone = next(
```

Comment:
You could rename this `new_poi` to express better the fact that we're actually creating a new object (based on an existing POI, but with custom coordinates).

Reply:
Done